src/styles.css

```diff
@@ -1 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
+body{
+    margin: 0;
+    background-color: #333333;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
```

Comment:
This comment applies to all files, try using Prettier extension to format your code properly. You can also use SCSS formatter extension to format scss files. This will help you avoid issues such as uneccessary spaces, wrong indentation, etc. 

Another suggestion is to use a Linter such as eslint. This will help you avoid errors such as empty constructor, empty lifecycle methods, no return type on methods, usage of vars/let, unused variables, etc